cli/dcos-spark/submit_builder.go

```diff
@@ -221,6 +221,10 @@ Args:
 	val.flag(submit).StringVar(&val.s)
 	args.stringVals = append(args.stringVals, val)
```

Comment:
Also you probably want to delete this line: https://github.com/mesosphere/spark-build/blob/2157a6f86b6edd49c455b8d9dffdcc85d84f8c27/cli/dcos-spark/submit_builder.go#L122

Reply:
I have deleted the line.